force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -625,6 +632,13 @@
         <shortDescription>Day</shortDescription>
         <value>Day</value>
     </labels>
+    <labels>
+        <fullName>List_View_Limit</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning when list view limit is reached.</shortDescription>
+        <value>The selected list view has 2000+ records, all matching records may not be displayed.</value>
```

Comment:
@lbeller - good morning, we have a warning message that will be displayed if the list view selected to filter the recent sessions component contains 2000+ records. Screen shot attached still pending UX approval. Thanks!
![Screen Shot 2020-12-11 at 8 44 25 AM](https://user-images.githubusercontent.com/450473/101916867-24471d80-3b8d-11eb-94d4-f65c2d6c3f17.png)